src/components/ui/topNavigation/topNavigation.component.tsx

```diff
@@ -158,7 +158,7 @@ export class TopNavigation extends React.Component<TopNavigationProps> {
         <View style={styles.leftControlContainer}>
           <FalsyFC component={accessoryLeft}/>
         </View>
-        <View style={alignmentStyles.titleContainer}>
+        <View style={[alignmentStyles.titleContainer, styles.titleContainer]}>
```

Comment:
@whitestranger7 consider using `||` operator, instead of merging to style objects. The idea here is that we want to apply one of 2 styles at a time according to our use case, but in alignment 'center' case they will be merged.